permissions/plugin-conjur-simple-integration.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "conjur-simple-integration"
+github: "jenkinsci/conjur-simple-integration"
```

Comment:
```suggestion
github: "jenkinsci/conjur-simple-integration-plugin"
```

Reply:
> 
Thanks for your update.
I did not realized that that was changed after I changed the name for the hosting request.